docs/specifications/tests/DNSQueryAndResponseDefaults.md

```diff
@@ -28,40 +28,40 @@ an explicit reference to this document.
 ## Default setting in *DNS Query*
 
 A *DNS Query* has the following default setting. A test case specification can
-refer to a *DNS Query* with one or several changes to the *Setting Items*
-overriding the default values. If a *Setting Item* is specified as "fixed" then
-the default value cannot be overidden.
+refer to a *DNS Query* with one or several changes to the *Parameters*
+overriding the default values. If a *Parameter* is specified as "fixed" (with an
+"X" in that column) then the default value cannot be overidden.
 
-|Setting Item |Default value |Fixed |Comment                       |
+|Parameter    |Default value |Fixed |Comment                       |
 |:------------|:-------------|:-----|:-----------------------------|
 |Protocol     | UDP          |      |                              |
-|OpCode       | "query"      | yes  |                              |
-|QR flag      | unset        | yes  | Always unset in a query      |
-|AA flag      | unset        | yes  |                              |
-|TC flag      | unset        | yes  |                              |
+|OpCode       | "query"      | X    |                              |
+|QR flag      | unset        | X    | Always unset in a query      |
```

Comment:
Why specify here in the comment that the parameter is unset since it is already fixed to `unset`?

Reply:
To underline that fact.

Reply:
It is just a comment. The choice of the other fixed, and default, values could also be commented, but I have not done that.

Reply:
Removed.